src/tools/pmempool/common.c

```diff
@@ -1450,8 +1450,9 @@ pool_set_file_open(const char *fname,
  * pool_set_file_close -- closes pool set file or regular file
  */
 void
-pool_set_file_close(struct pool_set_file *file)
+pool_set_file_close(struct pool_set_file **file_)
```

Comment:
IMHO it is not necessary at all. There is a lot of similar functions which does not clear the pointer.